lib/routes.js

```diff
@@ -1,41 +1,39 @@
-'use strict';
+'use strict'
 
-var Cache = require('./cache');
-var Winston = require('winston');
+const Cache = require('./cache')
+const Winston = require('winston')
 
-var EMPTY_TEMPLATE = '{"title":"","headers":[],"rows":[],"width":0,"height":0}';
-
-exports.facilityCodeLookup = function facilityCodeLookup(req, res, next) {
+const EMPTY_TEMPLATE = '{"title":"","headers":[],"rows":[],"width":0,"height":0}'
 
+exports.facilityCodeLookup = function facilityCodeLookup (req, res, next) {
   if (!req.query.criteria) {
-    Winston.info('Bad request: Missing query criteria');
-    return res.status(200).type('json').send(EMPTY_TEMPLATE);
+    Winston.info('Bad request: Missing query criteria')
+    return res.status(200).type('json').send(EMPTY_TEMPLATE)
```

Comment:
@BMartinos The idea was to mimic the response from the DHIS2 API. The cache returns the same response codes and values so that it's seamless to the upstream application as to whether or not the cache is being used.